mod_form.php

```diff
@@ -100,16 +102,27 @@ public function definition() {
         $users = get_enrolled_users($context, "mod/accredible:view", null, 'u.*');
 
         // Load final quiz choices.
-        $quizchoices = array(0 => 'None');
-        if ($quizes = $DB->get_records_select('quiz', 'course = :course_id', array('course_id' => $id) )) {
+        $quizchoices = array(0 => 'Select a Quiz');
+        if ($quizes = $DB->get_records_select('quiz', 'course = :course_id', array('course_id' => $id), '', 'id, name')) {
             foreach ($quizes as $quiz) {
                 $quizchoices[$quiz->id] = $quiz->name;
             }
         }
 
+        // Load course assigments.
+        $assigmentschoices = array(0 => 'Select an Activity Grade');
```

Comment:
Question - Would't it be better to use `array('' => 'Select an Activity Grade');` instead of `0`?